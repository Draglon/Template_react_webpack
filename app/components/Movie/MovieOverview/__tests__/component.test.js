import React from 'react';

import MovieOverviewComponent from '../component';

describe('<MovieOverviewComponent />', () => {
  const props = {
    title: 'some title',
    overview: 'some overview',
  };
  const component = shallow(<MovieOverviewComponent {...props} />);

  it('should match snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
