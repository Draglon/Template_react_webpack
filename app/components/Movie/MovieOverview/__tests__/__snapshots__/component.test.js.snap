// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MovieOverviewComponent /> should match snapshot 1`] = `
<Row
  className="top-margin"
  gutter={0}
>
  <Col
    offset={2}
    span={20}
  >
    <Title
      level={3}
    >
      some title
    </Title>
    <Paragraph
      className="movie__text"
    >
      some overview
    </Paragraph>
  </Col>
</Row>
`;
